kubernetes/omsagent.yaml

```diff
@@ -0,0 +1,686 @@
+        - operator: "Exists"
+          effect: "PreferNoSchedule"
+      volumes:
+        - name: docker-sock
+          hostPath:
+            path: /var/run
+        - name: container-hostname
+          hostPath:
+            path: /etc/hostname
+        - name: host-log
+          hostPath:
+            path: /var/log
+        - name: containerlog-path
+          hostPath:
+            path: /var/lib/docker/containers
+        - name: azure-json-path
+          hostPath:
+            path: /etc/kubernetes
+        - name: omsagent-secret
+          secret:
+            secretName: omsagent-secret
+        - name: omsagent-rs-config
+          configMap:
+            name: omsagent-rs-config
+        - name: settings-vol-config
+          configMap:
+            name: container-azm-ms-agentconfig
+            optional: true
+---
+  apiVersion: apps/v1
+  kind: DaemonSet
+  metadata:
+   name: omsagent-win
+   namespace: kube-system
+   labels:
+    component: oms-agent
+    tier: node
+  spec:
+   updateStrategy:
+    type: RollingUpdate
+   selector:
+    matchLabels:
+      component: oms-agent
+      tier: node
+   template:
+    metadata:
+      labels:
+        component: oms-agent
+        tier: node
+      annotations:
+        agentVersion: "1.10.0.1"
+        dockerProviderVersion: "10.0.0-1"
+        schema-versions: "v1"
+    spec:
+     serviceAccountName: omsagent
+     containers:
+       - name: omsagent-win
+         image: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod05262020-2"
+         imagePullPolicy: IfNotPresent
+         resources:
+          limits:
+           cpu: 150m
+           memory: 600Mi
+          requests:
+           cpu: 75m
+           memory: 225Mi
+         env:
+          # - name: AKS_RESOURCE_ID
+          #   value: "VALUE_AKS_RESOURCE_ID_VALUE"
+          # - name: AKS_REGION
+          #   value: "VALUE_AKS_RESOURCE_REGION_VALUE"
+          - name: ACS_RESOURCE_NAME
+            value: "my_acs_cluster_name"
+          - name: CONTROLLER_TYPE
+            value: "DaemonSet"
+          - name: HOSTNAME
+            valueFrom:
+               fieldRef:
+                 fieldPath: spec.nodeName
+         volumeMounts:
+          - mountPath: C:\ProgramData\docker\containers
+            name: docker-windows-containers
+            readOnly: true
+          - mountPath: C:\var #Read + Write access on this for position file
+            name: docker-windows-kuberenetes-container-logs
+          - mountPath: C:\etc\config\settings
+            name: settings-vol-config
+            readOnly: true
+          - mountPath: C:\etc\omsagent-secret
+            name: omsagent-secret
+            readOnly: true
+         livenessProbe:
+          exec:
+            command:
+              - cmd
+              - /c
+              - C:\opt\omsagentwindows\scripts\cmd\livenessProbe.cmd
+          periodSeconds: 60
+          initialDelaySeconds: 180
+     nodeSelector:

```

Comment:
https://msazure.visualstudio.com/DefaultCollection/CloudNativeCompute/_git/aks-rp/pullrequest/2921016?_a=files&path=%2Fccp%2Fcharts%2Fkube-control-plane%2Fcharts%2Fkube-addons%2Ftemplates%2Faddons%2F_omsagent.yaml

Line 432 : based on PR feedback we've switched to using nodeAffinity for both linux and windows (this doesn't have to be updated in this PR, you can also create a task for this in the next agent release user story imo)

Reply:
> https://msazure.visualstudio.com/DefaultCollection/CloudNativeCompute/_git/aks-rp/pullrequest/2921016?_a=files&path=%2Fccp%2Fcharts%2Fkube-control-plane%2Fcharts%2Fkube-addons%2Ftemplates%2Faddons%2F_omsagent.yaml
> 
> Line 432 : based on PR feedback we've switched to using nodeAffinity for both linux and windows (this doesn't have to be updated in this PR, you can also create a task for this in the next agent release user story imo)

> probably you should a create a task and ensure this is taken care of.